contributing/content-types.md

```diff
@@ -165,3 +166,26 @@ Quickstart guides are generally more conversational in tone than our other docum
 
 Examples
 TBD
+
+## Cookbook recipes
+The dbt Cookbook recipes are a collection of scenario-based, real-world examples for building with the dbt. Cookbook recipes offer practical, scenario-based examples for using specific features. 
+
+Most code examples are written in SQL, though you can apply the concepts in [Python](/docs/build/python-models) as well.
```

Comment:
Should we say that we prefer them in SQL unless they're targeting a specific Python use case?